app/models/user.rb

```diff
@@ -25,7 +25,7 @@ def initial_bitflyer_connection
   def average_purchased_price(product)
     product_purchases = purchases.where(product: product)
     return nil unless product_purchases.any?
-    return product_purchases.sum('price_at_purchase * shares') / product_purchases.sum(:shares)
+    return product_purchases.sum('price_at_purchase * shares') / product_purchases.sum(:shares).round(2)
```

Comment:
can we round this in the view rather than in the backend of stuff? (perhaps for all the numbers that we're dealing with)
I personally feel it's better to hand the views with the most accurate and raw information and we can render a cleaner version of it.